File: kotlin-runner/src/main/kotlin/code/sibyl/domain/sys/SysConfig.java
package code.sibyl.domain.sys;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.data.relational.core.mapping.Table;

import java.time.LocalDateTime;

/**
 * T_SYS_CONFIG
 */

@Entity
@Table("T_SYS_CONFIG")
@Data
@AllArgsConstructor
@NoArgsConstructor
public class SysConfig {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id", nullable = false)
    @org.springframework.data.annotation.Id
    private Long id;

    @Column(name = "NAME", nullable = false)
    private String name;

    @Column(name = "CODE", nullable = false)
    private String code;

    @Column(name = "CONTENT", nullable = false)
    private String content;

    @Column(name = "DESC", nullable = false)
    private String desc;

    @Column(name = "VERSION")
    private Long version;

    @Column(name = "is_deleted", nullable = false)
    private Boolean isDeleted;

    @Column(name = "create_time")
    private LocalDateTime createTime = LocalDateTime.now();

    @Column(name = "create_id")
    private Long createId;


    public void setId(Long id) {
        this.id = id;
    }

    public Long getId() {
        return id;
    }
}

File: kotlin-runner/src/main/kotlin/code/sibyl/domain/biz/SysConfig.kt
package code.sibyl.domain.biz

import jakarta.persistence.*
import lombok.Data
import org.springframework.data.relational.core.mapping.Table
import java.time.LocalDateTime

/**
 * 系统键值配置
 */
@Data
@Entity
@Table(name = "t_sys_config")
open class SysConfig : code.sibyl.domain.BaseEntity() {

    @Column(name = "key")
    var key: String? = null

    @Column(name = "value")
    var value: String? = null

    @Column(name = "type")
    var type: String? = null

    @Column(name = "name")
    var name: String? = null

    @Column(name = "description")
    var description: String? = null

    @Column(name = "is_system")
    var isSystem: Boolean? = null
}
